components/tikv_util/src/logger/file_log.rs

```diff
@@ -230,12 +193,118 @@ impl Rotator for RotateBySize {
     }
 }
 
+#[derive(Debug)]
+pub enum Task {
+    Archive,
+}
+
+impl Display for Task {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
+struct LogInfo {
+    f: DirEntry,
+    dt: DateTime<Local>,
+}
+
+fn dt_from_file_name(path: &Path, prefix: &str) -> Option<DateTime<Local>> {
+    let file_name_with_dt = path.file_stem().unwrap();
+    let mut dt = file_name_with_dt
+        .to_str()
+        .unwrap()
+        .to_string()
+        .replace(prefix, "");
+    if let Some(offset) = dt.find('T') {
+        unsafe {
+            // -2021-12-13T16-08-27.621 => -2021-12-13T16:08:27.621
+            let dt = dt.as_bytes_mut();
+            if dt.len() < offset + 6 {
+                return None;
+            }
+            dt[offset + 3] = b':';
+            dt[offset + 6] = b':';
+        };
+        dt.push_str("+00:00");
```

Comment:
The timezone seems not correct. The file is created using the local timezone, but always use UTC+0 here.

Reply:
This is intentional, because the time zone has been removed when the file name is generated, and it is added here only for time conversion. When we compare time, we use the +00:00 time zone uniformly

Reply:
OK, this is not a big problem. I was thinking that using the local timezone might be better.

Reply:
Changed to use the correct time zone, I also think it’s better